Sources/ArgumentParser/Usage/UsageGenerator.swift

```diff
@@ -369,6 +373,25 @@ extension ErrorMessageGenerator {
       return ""
     }
   }
+
+  func missingValueForOptionHelpMessage(name: Name) -> String {
+    guard let arg = arguments.first(where: { $0.names.contains(name) }) else {
+      return ""
+    }
+
+    let help = arg.help.abstract
+    return "\(name.synopsisString) <\(arg.valueName)> \(help)"
```

Comment:
Nitpick — here and below, can you add in one more space between the value name and the abstract?
```suggestion
    return "\(name.synopsisString) <\(arg.valueName)>  \(help)"
```

Reply:
done ✅